Client segfaults when listing VRF routes

With the latest gobgp release, listing routes in a VRF leads to a segfault of `gobgp` client.

# Steps to reproduce

1. Start a basic `gobgpd` instance, minimal config is enough

`/etc/gogbpd/gobgpd.yaml`
```yaml
global:
  config:
    as: 65000
    router-id: 172.16.0.1
    local-address-list:
      - 172.16.0.1
```

Started with: `./gobgpd -t yaml -f /etc/gobgpd/gobgpd.yaml`

2. Add a VRF: `gobgp vrf add blue rd 10.100:100 rt both 10.100:100`
3. Add a route in the VRF: `gobgp vrf blue rib add 10.0.0.0/24`
4. List routes in VRF: `gobgp vrf blue rib`

# Expected result

The VRF routes are listed:
```
   Network              Next Hop             AS_PATH              Age        Attrs
*> 10.0.0.0/24          0.0.0.0                                   00:00:02   [{Origin: ?}]
```

# Actual result

`gobgp` segfaults with the following stack trace:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x99f2a3]

goroutine 1 [running]:
main.showNeighborRib(0xb41be7, 0x3, 0x7ffccd4df7b0, 0x5, 0x0, 0x0, 0x0, 0x857624, 0xc000169590)
        /root/gobgp/cmd/gobgp/neighbor.go:947 +0x863
main.showVrf(0x7ffccd4df7b0, 0x5, 0xb72e50, 0xa22720)
        /root/gobgp/cmd/gobgp/vrf.go:119 +0x5b
main.newVrfCmd.func1(0xc00019cd80, 0xc0001a61a0, 0x1, 0x1)
        /root/gobgp/cmd/gobgp/vrf.go:199 +0x46
github.com/spf13/cobra.(*Command).execute(0xc00019cd80, 0xc0001a6170, 0x1, 0x1, 0xc00019cd80, 0xc0001a6170)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:653 +0x2af
github.com/spf13/cobra.(*Command).ExecuteC(0xc00019d680, 0xc0001a81a0, 0xc00012f760, 0xc0001a6040)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:728 +0x2cc
github.com/spf13/cobra.(*Command).Execute(0xc00019d680, 0xc00012f6d0, 0x1)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:687 +0x2b
main.newVrfCmd.func4(0xc00019d8c0, 0xc00012f6c0, 0x2, 0x2)
        /root/gobgp/cmd/gobgp/vrf.go:266 +0xd1
github.com/spf13/cobra.(*Command).execute(0xc00019d8c0, 0xc00012f600, 0x2, 0x2, 0xc00019d8c0, 0xc00012f600)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:653 +0x2af
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000b18c0, 0xc0001a3680, 0xc0001a38c0, 0xc0001a4000)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:728 +0x2cc
github.com/spf13/cobra.(*Command).Execute(0xc0000b18c0, 0x407290, 0xc00007c058)
        /root/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20170731170427-b26b538f6930/command.go:687 +0x2b
main.main()
        /root/gobgp/cmd/gobgp/main.go:33 +0x63
```

# Versions, software, ...

- OS: Debian 9 x64
- Go 1.11.1
- Gobgp `master` (commit 61d7a5e90bae220a45accf9914604ea568ae8c4a at the time of writing).

# Cause

After a bit of digging, it appears the `Destination` returned by the gRPC endpoint has a potentially incorrect `prefix` field. In the above case, this `prefix` has a value of `10.100:100:10.0.0.0/24`. Thus the call to `net.ParseCIDR` in `cmd/gobgp/neighbor.go:947` fails since this is not a valid IP with CIDR.

I don't know enough about the inners of `gobgpd` to say if the `prefix` field contains a valid value or not. It appears the contents of the `prefix` field is `{rd}:{actual prefix}`, which I don't know if it is right or not.

Oh and here is a quick patch to _at least_ be able to do stuff:
```patch
diff --git a/cmd/gobgp/neighbor.go b/cmd/gobgp/neighbor.go
index 90d9e4f8..779bba56 100644
--- a/cmd/gobgp/neighbor.go
+++ b/cmd/gobgp/neighbor.go
@@ -943,7 +943,8 @@ func showNeighborRib(r string, name string, args []string) error {
                        }
                        l := make([]*d, 0, len(rib))
                        for _, dst := range rib {
-                               _, p, _ := net.ParseCIDR(dst.Prefix)
+                               var tmp []string = strings.Split(dst.Prefix, ":");
+                               _, p, _ := net.ParseCIDR(tmp[len(tmp) - 1])
                                l = append(l, &d{prefix: p.IP, dst: dst})
                        }
```

--- a/cmd/gobgp/neighbor.go
+++ b/cmd/gobgp/neighbor.go
@@ -943,7 +943,12 @@ func showNeighborRib(r string, name string, args []string) error {
 			}
 			l := make([]*d, 0, len(rib))
 			for _, dst := range rib {
-				_, p, _ := net.ParseCIDR(dst.Prefix)
+				prefix := dst.Prefix
+				if t == api.TableType_VRF {
+					s := strings.Split(prefix, ":")
+					prefix = s[len(s)-1]
+				}
+				_, p, _ := net.ParseCIDR(prefix)
 				l = append(l, &d{prefix: p.IP, dst: dst})
 			}
 

